packages/components/Badge/src/Badge.tsx

```diff
@@ -10,45 +10,46 @@ import { Icon, createIconProps } from '@fluentui-react-native/icon';
 import { TextV1 as Text } from '@fluentui-react-native/text';
 
 import { stylingSettings } from './Badge.styling';
-import type { BadgeType, BadgeProps } from './Badge.types';
+import type { BadgeType, BadgeProps, BadgeState } from './Badge.types';
 import { badgeName } from './Badge.types';
 import { useBadge } from './useBadge';
 
-export const badgeLookup = (layer: string, userProps: BadgeProps): boolean => {
+export const badgeLookup = (layer: string, userProps: BadgeProps, state: BadgeState): boolean => {
   return (
     userProps[layer] ||
     layer === userProps['appearance'] ||
     layer === userProps['size'] ||
-    (!userProps['size'] && layer === 'large') ||
+    (layer === 'large' && !userProps['size']) ||
     layer === userProps['shape'] ||
-    (!userProps['shape'] && layer === 'circular') ||
+    (layer === 'circular' && !userProps['shape']) ||
     layer === userProps['badgeColor'] ||
-    (I18nManager.isRTL && layer === 'rtl')
+    (layer === 'rtl' && I18nManager.isRTL) ||
+    (layer === 'selected' && state && state.selected)
   );
 };
 
 export const Badge = compose<BadgeType>({
   displayName: badgeName,
   ...stylingSettings,
   slots: {
-    root: View,
+    root: Pressable,
```

Comment:
Could we make this change mobile only to reduce the impact of this change? I don't think this makes sense for dekstop.